### ISSUE TITLE
Req: Libraries from certain languages or colleciton tags

### ISSUE THREAD

**Issue by desimaniac** [adamgot/python-plexlibrary#32]
Would be nice addition. 

For example: Library for: Japanese Language + Animation Genre could go under _Anime Movies_. 

and / or

All movies with _Marvel Cinematic Universe_ tag could be turned into a Library.

**Comment by adamgot** [comment]
I have code for this in a separate script ready to be merged in, will hopefully get some time to do it soon.

**Comment by geekcroft** [comment]
@adamgot does that code include the Collection from genre bits you talked about months ago?

Also any closer to merging it in?

**Comment by adamgot** [comment]
@geekcroft No, I don't have any code for Collections, but it shouldn't be hard to do. What will take some time and effort is restructuring the existing codebase to not be centered around trakt-lists -> library.

**Comment by geekcroft** [comment]
@adamgot Sorry, half asleep this morning - I meant Library from genre :/
